Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14-alpine
```

Comment:
This is good, makes me wonder if the [autobuilder's node installation](https://github.com/mongodb/docs-worker-pool/blob/integration/Dockerfile#L24-L27) could be cleaned up.